src/edu/csus/ecs/pc2/core/imports/clics/CLICSJsonUtilities.java

```diff
@@ -0,0 +1,506 @@
+
+    /**
+     * Return a list of CLICS Awards for a contest.
+     * 
+     * @param contest
+     * @return
+     * @throws IOException
+     * @throws IllegalContestState
+     * @throws JAXBException
+     * @throws JsonMappingException
+     * @throws JsonParseException
+     */
+    public static List<CLICSAward> createAwardsList(IInternalContest contest) throws JsonParseException, JsonMappingException, JAXBException, IllegalContestState, IOException {
+        List<CLICSAward> list = new ArrayList<CLICSAward>();
+
+        Run[] runs = contest.getRuns();
+
+        if (runs.length == 0) {
+            return list;
+        }
+
+        addWinner(contest, list);
+
+        addFirstToSolve(contest, runs, list);
+
+        addGroupWinners(contest, runs, list);
+
+        addMedals(contest, list);
+
+        return list;
+    }
+
+    public static void addGroupWinners(IInternalContest contest, Run[] runs, List<CLICSAward> list) {
+
+        Arrays.sort(runs, new RunComparatorByElapsedRunIdSite());
+
+        Group[] groups = contest.getGroups();
+
+        if (groups.length == 0) {
+            // no groups defined, no group winners
+            return;
+        }
+
+        
+        /**
+         * This map contains the group/regional winner.  (The first team to solve a problem in a group/region)
+         */
+        Map<Group, ClientId> groupWinners = new HashMap<Group, ClientId>();
+        
+        try {
+            ContestStandings contestStandings = ScoreboardUtilites.createContestStandings(contest);
+
+            List<TeamStanding> teamStands = contestStandings.getTeamStandings();
+            for (TeamStanding teamStanding : teamStands) {
+                ClientId clientId = createClientId(teamStanding);
+                Group teamGroup = getGroupForTeam(contest, clientId);
+                if (teamGroup != null) {
+                    
+                    /**
+                     * Find group winner (or foundClientId will be null if no winner found yet)
+                     */
+                    ClientId foundClientId = groupWinners.get(teamGroup);
+                    
+                    if (foundClientId == null && isActive(contest,clientId)) {
+                        // found no group winner so if thie team is active it is the group winner
+                        
+                        if (!"0".equals(teamStanding.getSolved())) {
+                            /**
+                             * Team has to solve at least one problem to be the group winner 
+                             */
+                            groupWinners.put(teamGroup, clientId);
+                        }
+                    }
+                }
+            }
+        } catch (JAXBException | IllegalContestState | IOException e) {
+            StaticLog.getLog().log(Level.WARNING, "Error while calculating group winners DSA", e);
+        }
+
+        Set<Group> problemElementIds = groupWinners.keySet();
```

Comment:
The variable name `problemElementIds` here is pretty misleading; the `keyset` of `groupWinners` is a set of `Group`s; it has nothing to do with "problems".  Recommend refactoring the name of the variable to something more accurate, like `groupElementIds`.